solutions/USACO dec20-cowntagion.mdx

```diff
@@ -2,14 +2,76 @@
 id: usaco-1062
 source: USACO Silver 2020 December
 title: Cowntagion
-author: Kevin Sheng
+author: Kevin Sheng (Java, Python), Tanish Tyagi (C++)
 ---
 
 [Official Editorial](http://www.usaco.org/current/data/sol_prob1_silver_dec20.html)
 
+We can create a graph based off the roads between the farms which is given as input.
```

Comment:
This sentence feels unnecessary. We could delete or you can instead summarize the problem/solution or give some intuition?